doc/Comments.xml

```diff
@@ -297,6 +319,25 @@ Order(S5);
 ]]></Listing>
 </Subsection>
 
+
+<Subsection Label="@Listing">
+<Index Key="@Listing"><C>@Listing, @BeginListing, and @EndListing</C></Index>
+<Heading>@BeginListing <A>tested</A> and @EndListing</Heading>
+@BeginListing inserts an example into the manual. Opposed to @Example they have
+the same Syntax as GAPDoc examples and each line has to begin with a GAPDoc comment.
+@EndListing ends the example block.
+If <A>tested</A> is <C>false</C>, the example will not be tested if the manual examples are tested.
```

Comment:
The testing argument is already optional, but I will change it to not be a boolean.